### ISSUE TITLE
Improve Q scan behavior near gaps in the data

### ISSUE THREAD

**Issue by titodalcanton** [gwastro/pycbc#3221]
`pycbc_plot_qscan` makes confusing plots if it intersects the beginning or end of observing mode. One or more panels can be empty or show non-existing glitches. An example is visible [here](https://ligo.gravity.cf.ac.uk/~elisa.nitoglia/o3/runs/hl/c00/a40_initial/4._coincident_triggers/4.01_loudest_background_followup/). The plot could show a sentence describing the issue, or show some kind of hashed region in the data gap.

**Comment by GarethDaviesGW** [comment]
Is this solved by #3510 and/or #3511?

**Comment by titodalcanton** [comment]
Not really, I think this needs a bit more fiddling with `pycbc_plot_qscan`.
